site/content/docs/5.3/components/alerts.md

```diff
@@ -71,7 +71,7 @@ const alert = (message, type, typeVisuallyHidden) => {
     '   <p>',
     `     ${message}`,
     '   </p>',
-    '   <button type="button" class="btn-close" data-bs-dismiss="alert">',
+    '   <button type="button" class="btn-close" data-bs-dismiss="alert" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close">',
     '      <span class="visually-hidden">Close</span>',
     '   </button>',
     '</div>'
```

Comment:
Forgot about this until now but the function might be updated in the doc as well !